.rubocop.yml

```diff
@@ -57,7 +46,8 @@ Metrics/AbcSize:
   Description: >-
     A calculated magnitude based on number of assignments,
     branches, and conditions.
-  Enabled: false
+  Enabled: true
+  Max: 50
```

Comment:
I'm all for enabling `AbcSize`, we'll also need to add `Exclude` section with the list of files that violate the rule.

Reply:
@lightalloy I think the list of files will end up in the rubocop todo once we do enable this and regenerate the file, right?

Reply:
Yes, I just think modifying `rubocop_todo` where needed should be a part of this pr.

Reply:
ok perfect. @maestromac could you rebase this and regenerate the todo? So that we pickup the latest version of rubocop-todo you recently merged?

To regenerate the file you can do `rubocop --auto-gen-config`

Thank you!

Reply:
Updated.